manifests/nodejs.yml

```diff
@@ -135,7 +135,7 @@ tests/:
         Test_Scanners: v2.0.0
         Test_XSS: v2.0.0
       test_telemetry.py:
-        Test_TelemetryMetrics: missing_feature
+        Test_TelemetryMetrics: v4.16.1
```

Comment:
Most of the releases we make are minor versions, so I usually "preshot" the version number with a minor bump instead of a patch one